### PR TITLE
Kiriechem V1 and the chemical Researcher

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request

Creates a new system of randomized chemicals to discover every round and a new job, the Chemist, to take advantage of it

## Why It's Good For The Game
More neat things to make our server unique
This also removes strange reagent as a crafting recipe, fuck SR

## Changelog
:cl:
add: Added Randomized recipes
add: Added job to search for them, Chemical Researcher
del: Removed old SR recipe
tweak: Renames Strange Reagent to Verafore
tweak: Renames the old Chemist to Pharmacist.
/:cl:

### CODE REVIEW

**Comment by BootlegBow** [comment]
merg clonfick

File: ModularBungalow/Kiriechem/reagent.dm [hunk 1]
```diff
@@ -0,0 +1,67 @@
+//Kiriechem Chemicals
+/datum/reagent/medicine/bicardine //brings it back kinda
+	name = "Bicardine"
+	description = "A bruise reliever with a low overdose"
+	color = "#ECEC8D" // rgb: 236	236	141
+	taste_description = "spicy"
+	overdose_threshold = 6
```
Comment (BootlegBow) [inline ModularBungalow/Kiriechem/reagent.dm:7]: 6 is absurdly low for an overdose threshold

File: ModularBungalow/Kiriechem/reagent.dm [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	reagent_state = SOLID
+
+/datum/reagent/medicine/bicardine/on_mob_life(mob/living/M)
+	M.adjustBruteLoss(-2*REM)
+	..()
+	return TRUE
+
+/datum/reagent/medicine/bicardine/overdose_process(mob/living/M)
+	M.adjustFireLoss(3*REM)
```
Comment (BootlegBow) [inline ModularBungalow/Kiriechem/reagent.dm:16]: so taking bicardine _and_ kelotane at the same time allows you to bypass the overdose, that's prolly not good

File: ModularBungalow/Kiriechem/reagent.dm [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	..()
+
+
+/datum/reagent/medicine/kelotane //brings it back kinda
+	name = "Kelotane"
+	description = "A burn reliever with a low overdose."
+	color = "#ECEC8D" // rgb: 236	236	141
+	taste_description = "burning"
+	overdose_threshold = 6
```
Comment (BootlegBow) [inline ModularBungalow/Kiriechem/reagent.dm:25]: absurdly low overdose again

File: ModularBungalow/Kiriechem/reagent.dm [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+	reagent_state = SOLID
+
+/datum/reagent/medicine/kelotane/on_mob_life(mob/living/M)
+	M.adjustFireLoss(-2*REM)
+	..()
+	return TRUE
+
+/datum/reagent/medicine/kelotane/overdose_process(mob/living/M)
+	M.adjustBruteLoss(3*REM)
```
Comment (BootlegBow) [inline ModularBungalow/Kiriechem/reagent.dm:34]: same issue as bicardine

File: ModularTegustation/tegu_jobs/chemre/chemresearcher.dm [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "the research director, or the Chief medical officer, depending on which department you reside in."
+	selection_color = "#ffeeff"
+	exp_requirements = 60
+	exp_type = EXP_TYPE_CREW
+
+	outfit = /datum/outfit/job/chemre
+
+	access = list(ACCESS_MEDICAL, ACCESS_CHEMISTRY, ACCESS_RESEARCH, ACCESS_MINERAL_STOREROOM, ACCESS_RND)
```
Comment (BootlegBow) [inline ModularTegustation/tegu_jobs/chemre/chemresearcher.dm:14]: why do they need mineral storeroom access?
also later on we might wanna give the chem research labs their own access defines

**Review by BootlegBow — COMMENTED** [review]
i found the funnies

File: ModularBungalow/Kiriechem/reagent.dm (hunk 1 above)
Comment (Kitsunemitsu) [reply ModularBungalow/Kiriechem/reagent.dm:7]: That's by design

File: ModularBungalow/Kiriechem/reagent.dm (hunk 2 above)
Comment (Kitsunemitsu) [reply ModularBungalow/Kiriechem/reagent.dm:16]: Also by design, you have to FIND both and then take them.

File: ModularTegustation/tegu_jobs/chemre/chemresearcher.dm (hunk 5 above)
Comment (Kitsunemitsu) [reply ModularTegustation/tegu_jobs/chemre/chemresearcher.dm:14]: Mineral Storeroom to get things like silver for chemicals.

Comment (EgorDinamit) [reply ModularTegustation/tegu_jobs/chemre/chemresearcher.dm:14]: They can get any material from medical techfab already. No need to give them access to ORM.

**Comment by BootlegBow** [comment]
features are just intentional bugs

**Pull request merged**